spyder/plugins/editor/plugin.py

```diff
@@ -150,6 +150,8 @@ def __init__(self, parent, ignore_last_opened_files=False):
         self.dialog_size = None
 
         statusbar = parent.statusBar()  # Create a status bar
+        # Remove separator line
+        statusbar.setStyleSheet('QStatusBar::item {border: None;}')

```

Comment:
@dalthviz why was this necessary?

Reply:
This is needed to remove some white vertical lines between widgets that appear on Windows.

Reply:
Is this a QDarkstyle issue, or something from Spyder alone?

Reply:
Basically when using the dark theme the lines are more visible (since they are white). With the light theme the lines are also visible but are light gray:

![image](https://user-images.githubusercontent.com/16781833/64629108-39486600-d3b8-11e9-87d8-94e8051c9ac3.png)


![image](https://user-images.githubusercontent.com/16781833/64629001-07cf9a80-d3b8-11e9-9e5a-52bc94c7557f.png)

Reply:
Hmmmm ok 

@dalthviz could you submit this on Qdarkstyle repo, as this should not be the case by default (i think)